docs/release_notes/7.0.21-12.md

```diff
@@ -0,0 +1,42 @@
+
+## Release highlights
+
+This release provides the following features and improvements:
+
+### Boost performance during cluster restore and scaling with file copy based initial sync
+
+You can now select how a newly added or a restored replica set member receives the data from other members - via logical or file copy based sync. File copy based sync is the copying of physical files rather than actual data clone. This makes this sync method is faster than the logical one and it reduces your maintenance time on scaling.
+
+This functionality is available in [Percona Server for MongoDB Pro](../psmdb-pro.md) out of the box. Become a Percona Customer to enjoy all Pro features with little to no effort from your side.
+
+### Packaging changes
+
+Percona Server for MongoDB 7.0.21-12 is no longer supported on Ubuntu 20.04 (Focal Fossa) as this operating system has reached end of life. If you’re not ready to upgrade to a newer Ubuntu OS but still want to update Percona Server for MongoDB, contact us! We're here to make your databases run better.
+
+### Upstream Improvements
+
+The bug fixes, provided by MongoDB Community and included in Percona Server for MongoDB, are the following:
+
+* [SERVER-93120](https://jira.mongodb.org/browse/SERVER-93120) - Fixed the issue with blocking Full Time Diagnostic Data Capture (FTDC) collection when checking the state of the bacupCursor by using atomic mode instead of a lock
+* [SERVER-92806](https://jira.mongodb.org/browse/SERVER-92806) -  Tracked nested paths through MatchExpression trees while encoding indexability for plan cache entries
+* [SERVER-95672](https://jira.mongodb.org/browse/SERVER-95672) - Fixed the issue with missing documents when indexes on array fields contain subarrays by ensuring proper bounds when having nested arrays as predicates
+* [SERVER-97044](https://jira.mongodb.org/browse/SERVER-97044) -  Fixed an issue where change streams might incorrectly output a "drop" event during resharding or unsharding of a collection that is or was using zone sharding
+* [WT-12012](https://jira.mongodb.org/browse/WT-12012) - Fixed the issue when a generation drain process in WiredTiger, which manages eviction of older data, encounters an issue and then resolves itself when the verbosity levels for eviction and checkpoint operations increase. The issue is fixed by restoring original verbosity levels for these operations.
+
+Find the full list of changes in the release notes of [MongoDB 7.0.19 Community Edition](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.19---apr-14--2025) through [MongoDB 7.0.21 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/7.0/#7.0.21---upcoming).
+
+## Changelog
+
+### New Features
+
+* [PSMDB-1284](https://perconadev.atlassian.net/browse/PSMDB-1284) - Add the ability to perform file copy-based initial sync
+
```

Comment:
This release will also include PSMDB-1619, PSMDB-1633, PSMDB-1634

Reply:
Added